packages/vega-typings/types/spec/scale.d.ts

```diff
@@ -37,13 +37,18 @@ export type SortField =
     };
 
 /**
- * Unioned domains can only be sorted by count aggregate.
+ * Unioned domains can only be sorted by count, min, or max aggregates.
  */
 export type UnionSortField =
   | boolean
   | {
       op: 'count';
       order?: SortOrder;
+    }
+  | {
+      field: ScaleField;
+      op: 'count' | 'min' | 'max';
```

Comment:
Count ignores the field, right? Should we remove count then since we have the case above?

Reply:
The calculation ignores the field, but specifying a field for count is still legal. If we want to ban that option for multi-domain (but not other cases...) I suppose we could but I don’t see a strong reason to...